app/assets/stylesheets/themes/scinote.scss

```diff
@@ -1496,3 +1496,29 @@ html.turbolinks-progress-bar::before {
     width: 100px;
   }
 }
+
+// Only allow vertical resizing of all text areas
+textarea {
+  resize: vertical;
+}
+
+textarea.textarea-sm {
```

Comment:
This `textarea.textarea-sm` rule needs to be here because otherwise, the rule on `<textarea>`-s is overriden, and does not apply properly 😞 .